### ISSUE TITLE
Make article-specific yaml files for redirects and tags

### ISSUE THREAD

**Issue by cl8n** [ppy/osu-wiki#1780]
Current:
```
- Article_A/
- Article_B/
...
- redirect.yaml
```
Proposed:
```
- Article_with_redirects_or_tags/
    - properties.yaml
- Article_with_no_redirects_or_tags/
...
```
`properties.yaml` would contain two sequence maps, one for `redirects` and one for `tags`

needs a web implementation, of course. I think this would be easier to manage, since redirects and tags are not language-specific, and it's kinda annoying to remember about updating `redirect.yaml` every time an article is added/deleted/moved

**Comment by cl8n** [comment]
what is the purpose of `paths.yaml` in your suggestion?

**Comment by cl8n** [comment]
does anyone else have input on this

**Comment by Joehuu** [comment]
I agree, since it's hard to keep track of articles that have redirects/tags.

**Comment by Walavouchey** [comment]
my input on this is that it's easier to maintain one file than 900
